OPERATING.md

```diff
@@ -221,25 +188,141 @@ and `conversationId` is an identifier unique to transfer.
 
 ### Configuration
 In the event of an upload failure the adaptor will retry. By default, the retry limit 3 times. However, this is configurable 
-via the `STORAGE_RETRY_LIMIT`environment variable.
+via the `STORAGE_RETRY_LIMIT` environment variable.
 
 The adaptor requires permission to read, write and delete from the object storage bucket / container. For example, in AWS
 the adaptor would require permission to perform the actions `s3:GetObject`, `s3:PutObject` and `s3:DeleteObject`.
 
-Required environment variables:
-
-`STORAGE_TYPE`: The object storage type (S3, Azure or LocalMock) - Default = `localMock` - local development only
-`STORAGE_REGION`: AWS Specific (leave blank if using Azure)
-`STORAGE_CONTAINER_NAME`: Name of the storage bucket / container
-`STORAGE_REFERENCE`: The access key ID (AWS) or account name (Azure) - leave undefined if using an AWS instance role
-`STORAGE_SECRET`: The access key (AWS) or account key (Azure) - leave undefined if using an AWS instance role 
-
-Optional environment variables:
+The adaptor defaults to the `LocalMock` storage option which MUST NOT be used in production, and is designed only for
+testing. Attachments are not stored in a long term storage area when using `LocalMock`.
 
-`STORAGE_RETRY_LIMIT`: The amount of times to retry uploading the attachment before failing the transfer - Default = 3
+For more configuration see the [Attachment storage variables](#attachment-storage-variables) section.
 
 ## AWS daisy chaining example
 
+TODO
+
 ## Environment variables
 
+### Common variables
+
+#### PS queue variables
+
+These queue variables need to be the same between the translator and facade containers, as this is how the two services
+communicate.
+
+**Required**
+  - `PS_AMQP_BROKER`: the location of the PS Adaptors queue. This should be set to the url of a single JMS broker
+    (the PS Adaptor does not support concurrent PS Adaptor brokers), default = `amqp://localhost:5672`
+  - `PS_AMQP_USERNAME`: The username for accessing the PS broker
+  - `PS_AMQP_PASSWORD`: The password for accessing the PS broker
+
+**Optional**
+  - `PS_QUEUE_NAME`: The name of the patient switching queue, default = `pssQueue`
+  - `PS_AMQP_MAX_REDELIVERIES`: Number of times a message on the `PS_QUEUE_NAME` queue will be retried before being abandoned, default = `3`
+
+#### Logging configuration
+
+**Optional**
+  - `PS_LOGGING_LEVEL`: Spring logging level for facade use `DEBUG` for diagnosing problems in test environments, default = `INFO`
+
+#### DB
+
+**Required**
+  - `PS_DB_URL`: JDBC URL for Translator service, default = `jdbc:postgresql://localhost:5436`
+
+### Facade
+
+#### HTTP
+
+**Optional**
+  - `GPC_FACADE_SERVER_PORT`: HTTP server port for the [services endpoints](README.md#endpoints). 
```

Comment:
Do we need a default value of 8081 here?

Reply:
Good spot!